source/decryptor/decryptor.c

```diff
@@ -443,17 +490,14 @@ u32 NandPartitionsDumper() {
     u32 ctrnand_size;
     u32 keyslot;
 
-    switch (GetUnitPlatform()) {
-    case PLATFORM_3DS:
+    if(GetUnitPlatform() == PLATFORM_3DS) {
```

Comment:
To get rid of the compiler error. You could either initialize the variables with zeroes or do what I did. As what I did even reduces the length of the code, I deemed it the better option.